src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
+  max-width: 1024px;
```

Comment:
Better calculate max-width, for example, max-width: 100vw - 100vw is a CSS unit that stands for "100% of the viewport width, so if you need some specific width you can use function calc  - max-width: calc(100vw - 20px). This means that the width of the element will be equal to the full width of the viewport minus 20px.